File: AD-HOC/3093/3093.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int qtdTestes = scan.nextInt();
		
		while(qtdTestes-- > 0) {
			StringBuilder cartasNecessarias = new StringBuilder("QJKA");
			char[] cartas = scan.next().toCharArray();
			
			for(int i=0; i<cartas.length; i++) {
				for(int j=0; j<cartasNecessarias.length(); j++) {
					if(cartasNecessarias.charAt(j) == cartas[i]) {
						cartasNecessarias.deleteCharAt(j);
					}
				}
				if(cartas.length == 0) break;
			}
			
			if(cartasNecessarias.length() == 0) System.out.println("Aaah muleke");
			else System.out.println("Ooo raca viu");
		}
		scan.close();
	}
}

File: AD-HOC/3093/3093.cpp
#include <iostream>
#include <string>

using namespace std;

int main () {
	int qtdTestes;
	scanf("%d ", &qtdTestes);
	
	while(qtdTestes--) {
		string cartasNecessarias ("QJKA");
		string cartas;
		int total=0;
		
		getline(cin, cartas);
		
		for(int i=0; i<cartas.length(); i++) {
			for(int j=0; j<cartasNecessarias.length(); j++) {
				if(cartas[i] == cartasNecessarias[j]) {
					cartasNecessarias[j] = '0';
					total++;
				}
			}
		}
		
		if(total == 4) printf("Aaah muleke\n");
		else printf("Ooo raca viu\n");
	}
	return 0;
}
